docs/building_fledge/building_fledge.rst

```diff
@@ -561,10 +561,10 @@ In order to use the latest version for Fledge, add the following lines at the en
   export LD_LIBRARY_PATH=/usr/local/lib64
 
 
-Installing PostgreSQL 9.6
+Installing PostgreSQL 13 
 -------------------------
 
-CentOS provides PostgreSQL 9.2. Fledge has been tested with PostgreSQL 9.5, 9.6 and 10.X.
+CentOS provides PostgreSQL 9.2. Fledge has been tested with PostgreSQL 13, 9.5, 9.6 and 10.X.
```

Comment:
Replace this line with `Fledge has been tested with PostgreSQL 10.x and above.`

Reply:
fixed